fusor-ember-cli/app/routes/review/summary.js

```diff
@@ -4,7 +4,17 @@ export default Ember.Route.extend({
 
   model() {
     var deployment_id = this.modelFor('deployment').get('id');
-    return this.store.findRecord('deployment', deployment_id);
+    return Ember.RSVP.hash({
+      deployment: this.store.findRecord('deployment', deployment_id),
+      fusorDomainName: this.store.findAll('hostgroup').then(function(hostgroups) {
+        return hostgroups.filterBy('name', 'Fusor Base').get('firstObject')
+          .get('domain.name');
+      })
+    })
```

Comment:
I believe you need a semi colon on line 13. I got a build error when I was testing this.